code/DDSCodeTester.cpp

```diff
@@ -805,6 +805,17 @@ void dds_domain_examples()
             );
         //!--
     }
+
+    {
+        // IGNORE_LOCAL_ENDPOINTS_DOMAINPARTICIPANT
+        DomainParticipantQos participant_qos;
+
+        // Avoid local matching of this participant's endpoints
+        participant_qos.properties().properties().emplace_back(
```

Comment:
The `PropertyPolicyQos` is indeed implemented. It holds a 'dictionary' of sorts that the specific entity understands as it sees fit. In facto, this file is compiled as part of the documentation CI process.

It has two calls because the `PropertyPolicyQos` is a `fastrtps::rtps::PropertyPolicy`, which internally holds a `fastrtps::rtps::PropertySeq`, for which the getter happens to also be `properties()`. In sort, one to get the policy from the Qos and the other one to get the sequence from the policy.

If you look for other occurrences throughout the documentation you'll see it there too, e.g. [partitions](https://fast-dds.docs.eprosima.com/en/latest/fastdds/property_policies/non_consolidated_qos.html#endpoint-partitions).

Reply:
TY for the explanation and the reference!